src/ontology/mondo-edit.obo

```diff
@@ -330834,7 +330834,7 @@ synonym: "distal monosomy type 12p" EXACT [MONDORULE:4, Orphanet:280325]
 xref: ICD10:Q93.5 {source="ORDO:280325/attributed", source="ORDO:280325/ntbt", source="Orphanet:280325"}
 xref: Orphanet:280325 {source="MONDO:equivalentTo"}
 xref: UMLS:CN202714 {source="MONDO:equivalentTo"}
-is_a: MONDO:0017848 {source="Orphanet:280325"} ! partial deletion of the short arm of chromosome 12
```

Comment:
we shouldn't delete superclass assertions, we should make them excluded subclassOf

Reply:
oh wait, ignore me - this was the class that was obsoleted right? In that case, we don't need to exclude it.

Reply:
if this is the case, please go ahead and merge this, I will approve.